tools/xtask/src/build.rs

```diff
@@ -79,6 +85,78 @@ fn get_cli_configs(build_config: crate::BuildConfig, _other_options: &OtherOptio
     Ok(vec![configs])
 }
 
+fn build_third_party<'a>(
```

Comment:
For ARM I am currently avoiding compilation of all userspace components via the `-k/--kernel` flag. So we might need to add something like this to the beginning.
```rust
if !other_options.build_twizzler {
        return Ok(None);
}
```

Reply:
I am trying to build/test this right now, but try running `cargo start-qemu -ka aarch64 -m virt` and see if that works. @dbittman

Reply:
I added a test for this flag that just returns Ok(vec![]), should be fine.

Reply:
Just tried it and had not issues. It doesn't even look like it runs this step.